hack/new_version.sh

```diff
@@ -14,11 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-echo "Current skaffold config version: " $(sed -n 's;.*Version.*"skaffold/\(.*\)";\1;p' pkg/skaffold/schema/latest/config.go)
```

Comment:
Isn't this message useful when a version is not provided as an argument?

Reply:
I'll add it in `version.go`